### PR TITLE
Add community PR process documentation

### PR DESCRIPTION
**Pull request by trmartin4**
## 🎟️ Tracking

N/A

## 📔 Objective

Provide clarity for our community contributors around what our internal processes are for managing contributions. This is intended to be high-level enough that it doesn't have to change based on individual teams' implementation details around how they manage community PRs, but instead give contributors insight into what the communication points are likely to be when they submit a PR and help them understand why some PRs might take longer than others.

Once deployed, I intend to change the auto-generated message on all community PRs to include a link to this page,.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation
  team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: docs/contributing/pull-requests/community-pr-process.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+new features and offer up pull requests to resolve them.
+
+Transparency is one of our core values, and we feel that it is important that contributors
+understand the process that our teams take in evaluating, prioritizing, and reviewing these pull
+requests, and why some changes may move more quickly to completion than others.
+
+## Product assessment
+
+When your community contribution is received, the first step is for our Product team to evaluate
+whether the change aligns with the vision for the Bitwarden product. This is more relevant for
```
Comment (withinfocus) [inline docs/contributing/pull-requests/community-pr-process.md:21]:
```suggestion
whether the change aligns with the vision for the Bitwarden products. This is more relevant for
```

File: docs/contributing/pull-requests/community-pr-process.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+If the Product team has clarifying questions or would like the change to be modified to more closely
+align with their vision, they will reach out on the PR and request changes.
+
+Once Product has signed off on the change, it moves to our Engineering teams.
+
+## Engineering assessment
+
+The Engineering team(s) responsible for the change depend largely on what code was changed. We use
+Github code owners to manage the areas in our repositories owned by each team, and the responsible
```
Comment (withinfocus) [inline docs/contributing/pull-requests/community-pr-process.md:34]:
```suggestion
GitHub code owners to manage the areas in our repositories owned by each team, and the responsible
```

File: docs/contributing/pull-requests/community-pr-process.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+:::tip Assigned reviewers
+
+When a Bitwarden developer is assigned to your PR, that does not necessarily mean that the PR is
+under active review at that moment. This assignment may take place while the work is still on the
+team's backlog.
+
+:::
+
+The responsible Engineering teams will review the change and ensure that there are no concerns
```
Comment (withinfocus) [inline docs/contributing/pull-requests/community-pr-process.md:45]:
```suggestion
The responsible Engineering team(s) will review the change and ensure that there are no concerns
```

File: docs/contributing/pull-requests/community-pr-process.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+around how the change was implemented. This is generally more applicable for larger sets of changes,
```
Comment (withinfocus) [inline docs/contributing/pull-requests/community-pr-process.md:46]:
```suggestion
around how the change was implemented; this is generally more applicable for larger sets of changes,
```

**Review by withinfocus — APPROVED** [review]
Sorta-opinionated suggestions.

File: docs/contributing/pull-requests/community-pr-process.md (hunk 1 above)
Comment (trmartin4) [reply docs/contributing/pull-requests/community-pr-process.md]: Addressed with [`539ba43` (#471)](https://github.com/bitwarden/contributing-docs/pull/471/commits/539ba437c455b0df90d2c3b02dda0681cb87868d).

File: docs/contributing/pull-requests/community-pr-process.md (hunk 2 above)
Comment (trmartin4) [reply docs/contributing/pull-requests/community-pr-process.md]: [`539ba43` (#471)](https://github.com/bitwarden/contributing-docs/pull/471/commits/539ba437c455b0df90d2c3b02dda0681cb87868d)

File: docs/contributing/pull-requests/community-pr-process.md (hunk 3 above)
Comment (trmartin4) [reply docs/contributing/pull-requests/community-pr-process.md]: Addressed with [`539ba43` (#471)](https://github.com/bitwarden/contributing-docs/pull/471/commits/539ba437c455b0df90d2c3b02dda0681cb87868d).

File: docs/contributing/pull-requests/community-pr-process.md (hunk 4 above)
Comment (trmartin4) [reply docs/contributing/pull-requests/community-pr-process.md]: Addressed with [`539ba43` (#471)](https://github.com/bitwarden/contributing-docs/pull/471/commits/539ba437c455b0df90d2c3b02dda0681cb87868d).

**Review by withinfocus — APPROVED** [review]

**Pull request merged**